packages/web-sdk/src/instrumentations/performance/navigation.ts

```diff
@@ -23,17 +26,38 @@ export function getNavigationTimings(
       return;
     }
 
+    const navEntryJson = navigationEntryRaw.toJSON();
+
+    let spanContext: Pick<SpanContext, 'traceId' | 'spanId'> = {
+      traceId: '',
+      spanId: '',
+    };
+
+    if ('serverTiming' in navEntryJson) {
```

Comment:
This doesn't match with the data structure of `PerformanceServerTiming` wich is an object in the following structure (see [w3c docs](https://w3c.github.io/server-timing/#the-performanceservertiming-interface)).